formula-android/src/main/java/com/instacart/formula/integration/FragmentFlowRenderView.kt

```diff
@@ -130,6 +133,21 @@ internal class FragmentFlowRenderView(
         }
     }
 
+    private fun initializeFormulaIdIfNeeded(f: Fragment) {
```

Comment:
When a fragment is attached, we check if it has a generated id and generate one if it doesn't.

Reply:
TODO: maybe rename to `initializeFragmentIdIfNeeded`